vernac/himsg.ml

```diff
@@ -572,6 +572,8 @@ let rec explain_evar_kind env sigma evk ty =
       strbrk "the type of " ++ Id.print id
   | Evar_kinds.BinderType Anonymous ->
       strbrk "the type of this anonymous binder"
+  | Evar_kinds.EvarType evk ->
+      strbrk "the type of " ++ Evar.print evk
```

Comment:
Shouldn't we `pr_existential_key sigma evk` here instead?

Reply:
You're right, thanks a lot!